aggregator/src/main/java/com/redhat/cloud/notifications/DailyEmailAggregationJob.java

```diff
@@ -96,7 +96,11 @@ public void processDailyEmail() {
     }
 
     private boolean isCronJobEnabled() {
-        // The scheduled job is disabled by default.
-        return ConfigProvider.getConfig().getOptionalValue("notifications.aggregator.email.subscription.periodic.cron.enabled", Boolean.class).orElse(false);
+        final String cronJobEnabled = System.getProperty("NOTIFICATIONS_AGGREGATOR_EMAIL_SUBSCRIPTION_PERIODIC_CRON_ENABLED");
```

Comment:
If I'm not mistaken, the ConfigProvider will override `notifications.aggregator.email.subscription.periodic.cron.enabled` with `NOTIFICATIONS_AGGREGATOR_EMAIL_SUBSCRIPTION_PERIODIC_CRON_ENABLED` :) So I don't think we need this change.

Reply:
@josejulio but then I would expect that the new cron job should run here: https://console-openshift-console.apps.crcs02ue1.urby.p1.openshiftapps.com/k8s/ns/notifications-stage/pods/notifications-aggregator-27187200--1-8pk52/logs

Hope I am not overlooking something..

Reply:
How/Where are you defining `NOTIFICATIONS_AGGREGATOR_EMAIL_SUBSCRIPTION_PERIODIC_CRON_ENABLED` in there? I don't see it on the environment tab: https://console-openshift-console.apps.crcs02ue1.urby.p1.openshiftapps.com/k8s/ns/notifications-stage/pods/notifications-aggregator-27187200--1-8pk52/environment

Reply:
Forget about it.. my PR is still open... https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/24963